### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by Edoardo-Pedicillo**
This PR updates the documentation, considering #33, #34 and #26.

### CODE REVIEW

File: .github/workflows/build-docs.yml [hunk 1]
```diff
@@ -5,15 +5,19 @@ on:
   workflow_call:
     inputs:
         python-version:
+          description: 'used Python version'
           required: true
           type: string
         package-manager:
+          description: 'pip or poetry'
```
Comment (AleCandido) [inline .github/workflows/build-docs.yml:12]: A tiny note: choose one
1. all lower-case names (i.e. `python`, `pip`, and `poetry`)
2. all capitalized names (i.e. `Python`, `Pip`, and `Poetry`)
3. use the original case (i.e. [`python`](https://www.python.org/), [`pip`](https://pip.pypa.io/), and [`Poetry`](https://python-poetry.org/))

I don't like 2., but the others are both fine (3. is nicer, but more cumbersome to follow consistently, so the two things compensate).

File: .github/workflows/build-docs.yml [hunk 2]
```diff
@@ -5,15 +5,19 @@ on:
...
           required: true 
           type: string
         dependency-path:
+          description: 'Path to the dependency file'
```
Comment (AleCandido) [inline .github/workflows/build-docs.yml:16]: Use consistent capitalization: above you started with `used`, now with `Path`.

Personally, I slightly prefer `Path` (capitalize first word), but you decide. Just be consistent :)

**Review by AleCandido — COMMENTED** [review]
Extremely tiny review about parameters. I still have to review the md files.

Here I've been picky (it's something I see at first sight), but in general good job: it's exactly what I'd liked, i.e. brief descriptive comments :D

File: .github/workflows/build-docs.yml (hunk 1 above)
Comment (AleCandido) [reply .github/workflows/build-docs.yml:12]: Python is also cumbersome on its own: they always capitalize it in text, but the logo itself is lowercase...

Comment (Edoardo-Pedicillo) [reply .github/workflows/build-docs.yml:12]: I prefer the first one, because the inputs are all lower-case name

**Review by AleCandido — APPROVED** [review]

**Pull request merged**